HTML - 20/changeToCyrillic.js

```diff
@@ -0,0 +1,64 @@
+let latinLetters = document.getElementById("latin").value.toString().toLowerCase();

```

Comment:
You can remove "toString()."
This must be in changing function or it will be empty string always.